content/applications/finance/accounting/taxation/taxes/taxes.rst

```diff
@@ -1,3 +1,5 @@
+.. _Taxes:
```

Comment:
Also, when you need to add anchors, don't capitalize them, and prefix them so they don't risk colliding with other anchors sharing the same name. E.g, `Taxes` -> `accounting/taxation/taxes`.